services/notify/config.yml

```diff
@@ -12,17 +12,21 @@ defaults:
     maxMessageTime:               600
     # Email address to blacklist, useful to mitigate temporary misuse
     emailBlacklist:               !env:json EMAIL_BLACKLIST
```

Comment:
Let's plan to come back in a later PR and remove this support -- after we've gotten the UI working and transferred the current blacklisted addresses to it!

Reply:
Sounds good!